themes/digital.gov/layouts/partials/core/sitewide-alert.html

```diff
@@ -1,3 +1,4 @@
+// Creates sitewide banner alert at top of page under the banner
```

Comment:
In the future we should templatize this further by allowing the include to take a component variant, title, and body content. This way we can save some time in the future.

Reply:
I'll make a note of this and have you show me what you mean by this. I think that would be an excellent idea!

Reply:
Sure thing, I've created an issue with the feature request
https://github.com/GSA/digitalgov.gov/issues/5514